acct-user/vdr/vdr-0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="VDR (VideoDiskRecorder) service user"
+ACCT_USER_ID=452
+ACCT_USER_HOME=/var/lib/vdr
+ACCT_USER_HOME_OWNER=vdr:vdr
+# note: the first group name in the array will become the primary group
+ACCT_USER_GROUPS=( vdr audio cdrom video )
+IUSE="graphlcd legacy-homedir remote serial systemd"
+
+acct-user_add_deps
+
+pkg_setup() {
+	# if user wants to preserve his existing vdr installation, he can set
+	# USE=legacy-homedir
+	use legacy-homedir && ACCT_USER_HOME=/var/vdr
+
+	# media-plugins/vdr-graphlcd
+	use graphlcd && ACCT_USER_GROUPS+=( lp usb )
+
+	# media-plugins/vdr-remote, _only_ when systemd is installed
+	if use remote; then
+		use systemd && ACCT_USER_GROUPS+=( input ) || \
```

Comment:
I understand what you propose, but the logic I coded is exactly right: When using the plugin vdr.remote on a systemd.system the user vdr must be member of the group "input".
What you propose would lead into users forcing to migrate to systemd....

Reply:
I can't follow your logic. According to the ebuild, if someone enables `remote`, but not `systemd`, you will have a no-op, as it does nothing. This could lead to confusion.

Reply:
But you are exactly right. enable `remote` on non-systemd systems is not necessary and a no-op.
And remember: nobody should set `systemd` by hand, it comes with the selection of a systemd profile....

Reply:
If it's an invalid configuration, you should express that via REQUIRED_USE instead of an ewarn, ideally.

Reply:
@thesamesam The useflag `remote` on `systemd` systems is just a no-op, not invalid. I can change that to einfo, it is indeed not really a warning.